client/ctype.go

```diff
@@ -40,7 +40,7 @@ func (t CType) IsSupportedFieldCType() bool {
 func (t CType) IsCompatibleWith(kind FieldKind) bool {
 	switch t {
 	case PN_COUNTER, P_COUNTER:
-		if kind == FieldKind_NILLABLE_INT || kind == FieldKind_NILLABLE_FLOAT {
+		if kind == FieldKind_NILLABLE_INT || kind == FieldKind_NILLABLE_FLOAT64 || kind == FieldKind_NILLABLE_FLOAT32 {
```

Comment:
suggestion: An already untested behavior was changed, add a test if easy to do so.

Reply:
This should be covered already. I'll mark it as out of scope for now. and will revisit coverage later